admin/controllers/school.js

```diff
@@ -9,11 +9,7 @@ const ValidationError = require('../lib/validation-error')
 const errorConverter = require('../lib/error-converter')
 const hdfErrorMessages = require('../lib/errors/hdf')
 const hdfValidator = require('../lib/validator/hdf-validator')
-const {
-  fetchAnswers,
-  fetchScoreDetails,
-  fetchMultiplePupils,
-  fetchOnePupil } = require('../services/pupil.service')
+const pupilService = require('../services/pupil.service')
```

Comment:
@pris54 @jon-shipley @GeorgeChatzigiannis breaking down a service into function variables is not nice to work with.  Can we please ensure we require the entire service as a variable going forward.

Reply:
Also, this school controller is insanely long.  It needs breaking down in a big way.  I can see at least 2 other controllers such as results and attendance that could be made here.